libs/inspecjs/scripts/update_schema.sh

```diff
@@ -34,18 +20,15 @@ for SCHEMA in ${SCHEMAS[@]}
 do
     # Establish filenames
     OUTFILE="./src/generated_parsers/$VERSION/$SCHEMA.ts"
-    SCHEMAFILE="./work/schemas/$VERSION/$SCHEMA.json"
+    SCHEMAFILE="./schemas/$SCHEMA.json"
     MODIFIED_SCHEMAFILE="./work/tmp_$SCHEMA.json"
 
     # Loosen the schema
     COMPAT="./scripts/schema_compat_patches"
     cat $SCHEMAFILE | . "$COMPAT/generic.sh" | . "$COMPAT/$SCHEMA.sh" | . "./scripts/null_compat_schema.sh" > $MODIFIED_SCHEMAFILE
-
     # Generate the parser
     npx quicktype -l ts -s schema --src $MODIFIED_SCHEMAFILE -o $OUTFILE --runtime-typecheck
```

Comment:
I'm pretty sure using `npx` here is completely bypassing the version of quicktype that is installed and downloading a new version directly. This should just be `quicktype` without the npx prefix I believe.

Reply:
Right, I was meaning to ask you would it be the wrong move to use npx to run quicktype instead of downloading the latest version?

Reply:
Yes, the issue is using `npx` will always pull the latest version off of npm meaning that anytime a new version comes out CI could potentially start randomly failing for all PRs due to the new version.

Reply:
Gotcha, ok good to know, i'll make the change